phys2denoise/cli/run.py

```diff
@@ -0,0 +1,196 @@
+                               'Default is current folder.',
+                          default='.')
+    optional.add_argument('-sr', '--sample-rate',
+                          dest='sample_rate',
+                          type=float,
+                          help='Sampling rate of the physiological data in Hz.',
+                          default=None)
+    optional.add_argument('-pk', '--peaks',
+                          dest='peaks',
+                          type=str,
+                          help='Filename of the list with the indexed peaks\' positions'
+                               ' of the physiological data.',
+                          default=None)
+    optional.add_argument('-thr', '--throughts',
+                          dest='throughts',
+                          type=str,
+                          help='Filename of the list with the indexed peaks\' positions'
+                               ' of the physiological data.',
+                          default=None)
+    optional.add_argument('-os', '--oversampling',
+                          dest='oversampling',
+                          type=int,
+                          help='Temporal oversampling factor in seconds. '
+                               'Default is 50.',
+                          default=50)
+    optional.add_argument('-tl', '--time-length',
+                          dest='time_length',
+                          type=int,
+                          help='RRF Kernel length in seconds.',
+                          default=None)
+    optional.add_argument('-onset', '--onset',
+                          dest='onset',
+                          type=float,
+                          help='Onset of the response in seconds. '
+                               'Default is 0.',
+                          default=0)
+    optional.add_argument('-tr', '--tr',
+                          dest='tr',
+                          type=float,
+                          help='TR of sequence in seconds.',
+                          default=None)
+    optional.add_argument('-bts', '--belt-ts',
+                          dest='belt_ts',
+                          type=str,
+                          help='Filename of the 1D array containing the .'
+                               'respiratory belt time series.',
+                          default=None)
+    optional.add_argument('-win', '--window',
+                          dest='window',
+                          type=int,
+                          help='Size of the sliding window in seconds. '
+                               'Default is 6 seconds.',
+                          default=6)
+    optional.add_argument('-osr', '--out-samplerate',
+                          dest='out_samplerate',
+                          type=float,
+                          help='Sampling rate for the output time series '
+                               'in seconds. Corresponds to TR in fMRI data.',
```

Comment:
Isn't this a duplicate of `--tr` then?

Reply:
We could rename the variable in the metrics that use this one